.eslintrc.js

```diff
@@ -9,6 +9,18 @@ module.exports = {
   rules: {
     'prettier/prettier': 'error',
     'unused-imports/no-unused-imports': 'error',
+    'no-restricted-imports': [
```

Comment:
Nice thanks for adding this failsafe :D

Reply:
Yeah, thought it was a good addition just to be safe that we don't accidentally import `isWeekend` from `date-fns` and expect it to work differently.
Actually surprise that `isWeekend` from `date-fns` didn't have this feature...

Reply:
Even worse they have it on `startOfWeek` util 🙈